src/ResponsiveProvider.js

```diff
@@ -81,56 +72,81 @@ const ResponsiveProvider = ({
         };
     }, [breakpointsRef]);
 
-    // Add orientation media query
+    return { isCalculated, currentMediaType };
+};
+
+const useDebugResponsive = (contextObject, currentMediaType) => {
     useEffect(() => {
-        const setMatchedOrientation = (matches) => {
-            setCurrentOrientation(matches ? 'portrait' : 'landscape');
-        };
+        if (!process || !process.env || process.env.NODE_ENV !== 'test') {
+            /* eslint-disable no-console */
+            console.group(
+                '%c @farfetch/react-context-responsive updated!',
+                'color:#c4aa84; font-weight:bold;'
+            );
+            console.log(
+                '%c Current media type:',
+                'font-weight:bold;',
+                currentMediaType
+            );
+            console.log('%c Context:', 'font-weight:bold;', contextObject);
+            console.groupEnd();
+            /* eslint-enable no-console */
+        }
+    }, [contextObject, currentMediaType]);
+};
 
-        const orientationMediaQueryList = window.matchMedia(
-            '(orientation: portrait)'
+const ResponsiveProvider = ({
+    initialMediaType,
+    defaultOrientation,
```

Comment:
Can we rename this to initialOrientation? We have `initialMediaType` that as the same objective. I think we should use or `default` or `initial` prefix.

What do you think?

Reply:
Yea, I thought about this as well, but for some reason "default" sounds better on orientation.  There is actually an inconsistency in React itself, for the useState hook here https://reactjs.org/docs/hooks-reference.html#usestate , they call it initialState and so use the initial prefix, however for props, they call it defaultProps ...

I would like to do a bit more research and look into what other APIs use and if there is some logic behind how they choose one or the other.  

Still though, this is a breaking change, and I didn't want to do a breaking change in something that is supposed to be a refactor.  There are some other API changes I would like to make, but let's see what @SoaresMG and the other maintainers think and we can maybe consider batching this breaking change along with the others.  What do you think?

Reply:
Yeah, we should think about not making breaking changes for now